### PR TITLE
Create setup.md

### PR DESCRIPTION
**Pull request by baileythegreen**
Add information about Jekyll setup.

@fmichonneau  We should maybe discuss the CSS styling for this.

### CODE REVIEW

**Review by fmichonneau — APPROVED** [review]

File: setup.md [hunk 1]
```diff
@@ -0,0 +1,255 @@
+---
+title: Setup
+---
+
+Our lesson template is kept in the [`carpentries/styles` repository][styles]. The `styles`
```
Comment (fmichonneau) [inline setup.md:5]: we probably don't want to include this paragraph

File: setup.md [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+repository is carefully curated so that changes made to it are easily mergeable by downstream
+lessons. The `styles` repository contains various bits that take Markdown files and render them as a
+lesson web page. For more information on how to develop lessons and maintain them, see our
+[lesson-example][lesson-example]. It will walk you through the basics of lesson design and how to
+use GitHub, Markdown and Jekyll for lesson development. Follow the instructions below to make your
+own empty lesson in your own GitHub account. Once you've done that you can just write Markdown code
+and have lesson web pages just like the [lesson-example][lesson-example] and all of our other
+lessons, but with your lesson content.
```
Comment (baileythegreen) [inline setup.md:13]:
```suggestion
```

**Comment by baileythegreen** [comment]
I've removed the unnecessary text.

**Comment by fmichonneau** [comment]
Thank you!

**Pull request merged**